### ISSUE TITLE
fatal error

### ISSUE THREAD

**Issue by braxhunter** [Aussiedroid/AD-EDWarthogEnhancedScript#19]
Connecting virtual keyboard...Done
Connecting virtual mouse (absolute axes)...Done

!! FATAL ERROR !!
EXITING Elite Dangerous .. Reason: Status.json file *Not Found*
Please set 'StatusFile' alias correctly in User Preferences
Refer to AD_EDUserPrefs.tmh file for instructions. Closing Down Script...
Script requested Abort!
Aborting script (check output above for reason)...
Script stopped!

Runtime Error: Script run aborted in main ( line 218 in AD_EDStatusJson_v5.0.0.tmh )

**Comment by braxhunter** [comment]
sorry... did not see the comment to change... you should actually put in the suggested code instead of the z:\\ etc... confusing..

**Comment by Aussiedroid** [comment]
Thanks braxhunter,

Good to see the error message is working (I had not tested that yet!). 

When I complete my release version, I will update it to the more default path on C:\ .. just left as Z:\ for now as that is my personal location. 

Have put a callout to it in the Changelog too .. so hopefully wont catch too many CMDRs out :)

Thanks for checking out the Odyssey build too .. please let me know if there are any issues with the MFD code I added.

Cheers,
AD

**Comment by braxhunter** [comment]
no problem, i am teting is out now, but because we cannot fly in odyssy its hard to test but I did run into errors switch the HATs for controlling the pips and menus.. they did not allow me to switch. I will test the version 5.0 in the horizons to see if there are any errors. I am almost done with the tutorial for the MFD's and I added some mappings and updated the graphic file. I will send you those tomorrow
